realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/ProcessCommitTests.java

```diff
@@ -52,10 +52,9 @@
 @RunWith(AndroidJUnit4.class)
 public class ProcessCommitTests extends BaseIntegrationTest {
 
-    // FIXME: Ignore for now. They do still not work. It might be caused by two processes each creating
     // a Sync Client, but it needs to be investigated.
     @Test
-    @Ignore
+    @Ignore("Ignore for now. Still doesn't work. It might be caused by two processes each creating")
```

Comment:
I have no idea what's going on here.  I just explicitly ignored the test, instead of commenting it out.